.changeset/popular-keys-fetch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': minor
```

Comment:
Should we make it a patch?

Reply:
I thought about it and I believe this is a feature. We’re _adding_ compatibility for a feature, not fixing an existing one.